src/tictactoe_server/core.clj

```diff
@@ -1,7 +1,14 @@
 (ns tictactoe-server.core
-  (:gen-class))
+  (:gen-class)
+  (:require [webserver.app :as app]
+            [tictactoe-server.app :as backing-app]
+            [tictactoe-server.new]
+            [tictactoe-server.move]))
 
-(defn -main
-  "I don't do a whole lot ... yet."
-  [& args]
-  (println "Hello, World!"))
+(defn -main [& args]
+  (let [server (java.net.ServerSocket. 5000)
```

Comment:
It's usually nice to be able to pass in a port number incase there is a conflict with something else